includes/class-lipscore-product-helper.php

```diff
@@ -68,6 +68,11 @@ protected function product_gtin( $product ) {
             $gtin = get_post_meta($product->get_id(), $gtin_attr, true);
         }
 
+        if($gtin){
+
+            $gtin = explode( ', ', $gtin );
```

Comment:
Just tiny code style fixes:
please remove empty line above `$gtin = explode( ', ', $gtin );`
and add spaces `if ( $gtin ) {`

https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#space-usage